### PR TITLE
Add add and dob to ccs records and child health records

### PR DESCRIPTION
**Pull request by emord**
@sheelio @dannyroberts Adds the values to the UCR table we talked about the other day. Will still need to follow it up with a change to the aggregation script, but want to get this out sooner to get the data there.

budyd @proteusvacuum

### CODE REVIEW

**Review by sheelio — APPROVED** [review]

File: custom/icds_reports/ucr/data_sources/ccs_record_cases_monthly_tableau2.json [hunk 1]
```diff
@@ -1760,6 +1760,23 @@
...
+        "display_name": null,
+        "expression": {
+          "expression": {
+            "datatype": null,
+            "property_name": "add",
+            "type": "property_name"
+          },
+          "type": "root_doc"
+        },
+        "is_nullable": false,
```
Comment (sheelio) [inline custom/icds_reports/ucr/data_sources/ccs_record_cases_monthly_tableau2.json]: Actually, shouldn't this be nullable = true?

**Review by sheelio — CHANGES_REQUESTED** [review]

File: custom/icds_reports/ucr/data_sources/ccs_record_cases_monthly_tableau2.json (hunk 1 above)
Comment (emord) [reply custom/icds_reports/ucr/data_sources/ccs_record_cases_monthly_tableau2.json]: Oh. wow. yes. thanks. Fixed this

**Review by sheelio — APPROVED** [review]

**Review by dannyroberts — APPROVED** [review]
gnarly test changes! Looks good.

**Comment by emord** [comment]
> gnarly test changes! Looks good.

Yeah, just adding the columns at the moment. I'm planning  re-do these for the next version of the dashboard because they're nasty

**Pull request merged**